### PR TITLE
Add ProjectAPI improvement to release note

### PR DESCRIPTION
**Pull request by Ibaqu**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -683,6 +683,15 @@ Added runtime breakpoint verification support. With this improvement, the debugg
 - Added support to validate the Ballerina resource headers with OpenAPI operation headers. With this improvement, the validator gives validation errors on missing headers, undocumented headers, and type mismatch of the header parameters.
 - Added support to validate the Ballerina resource return type with OpenAPI operation response. With this improvement, the validator gives validation errors on missing return status codes, missing return payload types, undocumented return status codes, and undocumented payload media types.
 
+## Ballerina packages updates
+
+## Improvements
+
+### Improved Package resolution
+
+- Added support to specify the dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:692]:
```suggestion
- Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.
```

Comment (azinneera) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:692]: Also, shall we add an example?

Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Corrected description and added an example

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
 - Added support to validate the Ballerina resource headers with OpenAPI operation headers. With this improvement, the validator gives validation errors on missing headers, undocumented headers, and type mismatch of the header parameters.
 - Added support to validate the Ballerina resource return type with OpenAPI operation response. With this improvement, the validator gives validation errors on missing return status codes, missing return payload types, undocumented return status codes, and undocumented payload media types.
 
+## Ballerina packages updates
+
+## Improvements
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:688]:
```suggestion
### Improvements
```

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+
+### Improved Package resolution
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:690]: We don't need this IMO

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+
+- Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:692]:
```suggestion
- Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.

For example, the minimum version of `ballerinax/mysql` dependency can be specified as shown below.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+```toml
+[[dependency]]
+org = "ballerina"
+name = "io"
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:696]:
```suggestion
name = "mysql"
```

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 6]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+version = "1.2.0”
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:697]:
```suggestion
version = "1.3.0”
```

Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Changed version

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 5 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Changed package name

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 7]
```diff
@@ -683,6 +683,21 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+## Ballerina packages updates
+
+## Improvements
+
+### Improved Package resolution
+
+- Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.
+```toml
+[[dependency]]
+org = "ballerina"
```
[the remark this answers is not in the document]
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Changed org

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 4 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Added example desc

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 3 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Removed `Improved Package Resolution`

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 2 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Corrected title

**Review by azinneera — APPROVED** [review]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 8]
```diff
@@ -683,6 +683,20 @@ Added runtime breakpoint verification support. With this improvement, the debugg
 - Added support to validate the Ballerina resource headers with OpenAPI operation headers. With this improvement, the validator gives validation errors on missing headers, undocumented headers, and type mismatch of the header parameters.
 - Added support to validate the Ballerina resource return type with OpenAPI operation response. With this improvement, the validator gives validation errors on missing return status codes, missing return payload types, undocumented return status codes, and undocumented payload media types.
 
+## Ballerina packages updates
+
+### Improvements
+Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the version specified or the latest compatible version.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:699]:
```suggestion
Added support to specify the minimum required dependency version in the `Ballerina.toml` file. With this improvement, the dependencies will get updated to the specified version or the latest compatible version.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md [hunk 9]
```diff
@@ -683,6 +683,20 @@ Added runtime breakpoint verification support. With this improvement, the debugg
...
+
+For example, the minimum version of `ballerinax/mysql` dependency can be specified in the following way.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md:701]:
```suggestion
For example, the minimum version of the `ballerinax/mysql` dependency can be specified in the following way.
```

**Review by praneesha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 8 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Addressed

File: downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md (hunk 9 above)
Comment (Ibaqu) [reply downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md]: Addressed

**Pull request merged**